NucleonSecurity/nucleonedr/ingest/parser.yml

```
@@ -0,0 +1,128 @@
+name: edr-ecs-core-basic
+ignored_values: null
+
+pipeline:
+  - name: json
+    external:
+      name: json.parse-json
+  - name: ts
+    external:
+      name: date.parse
+      properties:
+        input_field: json.message.timestamp
+        output_field: ts
+        timezone: UTC
+  - name: base
+  - name: exec_stage
+  - name: net_stage
+  - name: file_read
+  - name: file_change
+  - name: file_delete
+  - name: openproc_stage

```

From other comments:
```suggestion
  - name: exec_stage
    filter: "{{ json.message.type == 'execute' }}"
  - name: net_stage
    filter: "{{ json.message.type == 'network' }}"
  - name: file_read
    filter: "{{ json.message.type == 'read' }}"
  - name: file_change
    filter: "{{ json.message.type in ['write','rename'] }}"
  - name: file_delete
    filter: "{{ json.message.type == 'delete' }}"
  - name: openproc_stage
    filter: "{{ json.message.type == 'open_process' }}"
```